operator/documentation/asciidoc/proc_running_quickstart.adoc

```diff
@@ -0,0 +1,61 @@
+= Storing and Retrieving Data on {ProductName} Clusters
+
+.Prerequisites
+
+* Complete the procedures to create a {FullProductName} cluster with the
+{ispn_operator} and then retrieve credentials for the application user. See the
+following topics in the {ProductName} documentation:
+
+- link:https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.3/html-single/running_the_data_grid_operator/index#spinning_up_clusters[Spinning Up Data Grid Clusters]
+- link:https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.3/html-single/running_the_data_grid_operator/index#getting_credentials-cluster_auth[Retrieving Cluster Credentials]
+
+.Procedure
+
+. Expose the {ProductName} service for external access.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ oc expose svc example-rhdg
```

Comment:
Where is `example-rhdg` coming from? The example Infinispan yaml would need to be modified for this. We've not yet made such changes for product.

Reply:
`example-rhdg` is the example name for the pods that we use in the product docs at: https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.3/html/running_the_data_grid_operator/spinning_up_clusters#creating_minimal_clusters-create_cluster

I actually want to go back and test this again now...

Reply:
@galderz took me long enough to get crc working but I've tested this against the product docs in the previous comment.

`example-rhdg` is in the productized cr yaml files. it looks like the Operator creates the service based on the for the pods in the cluster.

Reply:
Ok, I guess it didn't seem clear enough to me that the user needs to complete the steps before. Me half-reading ;)